doc/github_handler.rst

```diff
@@ -0,0 +1,60 @@
+GitHub Handler
+==============
+
+GitHub Handler is a Universum mode that serves as GitHub Application, helping to perform and report checks on
+new commits to a repository. In this mode Universum can create new check runs on GitHub and trigger an already
+set up automation server to perform these checks. GitHub Handler parses all required params and passes them
```

Comment:
We should clearly separate main mode and handler mode in the text. In this and previous line two different names for the same thing are used: "this Universum mode" and "GitHub Handler". While technically this is correct, it is confusing for the reader.

I propose to either never call handler "a mode" or always refer to it as "GitHub handler mode".